Makefile

```diff
@@ -147,40 +160,45 @@ ifeq ($(TARGET_PLATFORM),linux)
 endif
 
 release/$(APPLICATION_NAME)-$(TARGET_PLATFORM)-$(TARGET_ARCH)-rw.dmg: \
-	release/$(APPLICATION_NAME)-darwin-$(TARGET_ARCH)
+	release/$(APPLICATION_NAME)-darwin-$(TARGET_ARCH) \
+	| release
 	./scripts/darwin/electron-create-readwrite-dmg.sh -p $< -o $@ \
 		-n "$(APPLICATION_NAME)" \
 		-i assets/icon.icns \
 		-b assets/osx/installer.png
 
+release/out: | release
+	mkdir -p $@
```

Comment:
Sure. The intention was to have it near the rules that actually use `release/out`, but I like the suggestion.